dashboard/app/controllers/plc/enrollment_task_assignments_controller.rb

```diff
@@ -0,0 +1,35 @@
+class Plc::EnrollmentTaskAssignmentsController < ApplicationController
```

Comment:
No auth for this model?  If you add auth, you can use `load_and_authorize_resource` to get rid of `set_plc_enrollment_task_assignment` (same pattern in `Plc::CoursesController`).